cvmfs/cvmfs_config

```diff
@@ -417,7 +422,8 @@ cvmfs_chksetup() {
   fi
 
   # Check SELinux label of cache directory
-  if [ -f /selinux/enforce ] && [ $(cat /selinux/enforce) -ne 0 ]; then
+  if has_selinux && which semodule > /dev/null 2>&1 && \
+     semodule --list-modules 2> /dev/null | grep -q cvmfs; then
```

Comment:
The old check didn't work on Fedora (because the `/selinux` directory is not available there).